charts/dify/values.yaml

```diff
@@ -39,6 +39,8 @@ global:
   #     secretKeyRef:
   #       name: dify
   #       key: SECRET_KEY
+  - name: STORAGE_TYPE
+    value: "s3"
```

Comment:
This is a breaking change for people who have already overridden `extraBackendEnvs` and using `s3` as storage backend.

Please render this environment variable and set the variable value with `global.storageType` like the `global.edition` environment.

Reply:
For example, if people using this value file trying to upgrade chart
```
global:
  extraBackendEnvs:
  - name: foo
    value: bar
```

The storage_type environment would not be present for deployments:

```
@LeoQuote ➜ /workspaces/charts (master) $ helm template dify charts/dify -f dify-value.yaml | grep -C 10 STORAGE_TYPE 
```

Reply:
Thanks for feedback! I've done some changes but I don't get it why it should not be present in deployments? This variable is already present before my changes, also `EDITION` env is present.

Reply:
`global.extraBackendEnvs` is a list, when overriding this value, helm would use the user provided value instead of merging the list.